File: Soving/BAEKJOON/bronze/B2_2920.py
li = list(map(int, input().split()))

a = li[0]
answer = 'mixed'
if a == 8:
    for l in li[1:]:
        if a -1 == l:
            a = l
        else:
            break
    else:
        answer = 'descending'
elif a == 1:
    for l in li[1:]:
        if a +1 == l:
            a = l
        else:
            break
    else:
        answer = 'ascending'
print(answer)

File: java/backjoon/class/class1/B2_2920.java
import java.io.FileInputStream;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class B2_2920 {
    public static void main(String[] args) throws FileNotFoundException {
        System.setIn(new FileInputStream("input.txt"));
        Scanner sc = new Scanner(System.in);
        int[] pitchs = new int[8];
        pitchs[0] = sc.nextInt();
        if (pitchs[0]==1 || pitchs[0]==8) {
            System.out.println(solution(pitchs, sc));
        }
        else {
            System.out.println("mixed");
        }

    }

    public static String solution(int[] pitchs, Scanner sc) {
        int flag;
        if(pitchs[0] == 1){
            flag = 1;
        }
        else {
            flag = -1;
        }
        for (int i=1; i<8; i++) {
            pitchs[i] = sc.nextInt();
            if (pitchs[i-1]+flag != pitchs[i]) {
                return "mixed";
            }
        }
        if (flag==1) {
            return "ascending";
        }
        return "descending";
    }
}
